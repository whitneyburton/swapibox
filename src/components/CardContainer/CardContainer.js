import React from 'react';
import './CardContainer.scss';
import { Card } from '../Card/Card';

export const CardContainer = ({ category, people, planets, vehicles }) => {
  
  const allPeople = people.map(person => {
    return <Card
      key={person.name}
      person={person}
    />;
  });

  return (
    <div className='CardContainer'>
      { allPeople }
    </div>
  )
}