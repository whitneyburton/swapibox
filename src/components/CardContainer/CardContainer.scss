.CardContainer {
  grid-area: cardcontainer;
  padding: 0px 10px;
  height: 79vh;
  display: grid;
  grid-template-columns: 1fr 1fr 1fr;
  grid-template-rows: max-content;
  overflow: scroll;
}