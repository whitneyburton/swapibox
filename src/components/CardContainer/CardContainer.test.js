import React from 'react';
import { CardContainer } from './CardContainer';
import { shallow } from 'enzyme';

describe('CardContainer', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<CardContainer />)
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
})