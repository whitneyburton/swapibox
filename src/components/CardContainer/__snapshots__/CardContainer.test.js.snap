// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardContainer should match the snapshot 1`] = `
<div
  className="CardContainer"
>
  <h2>
    CardContainer
  </h2>
  <Card />
</div>
`;
